.github/workflows/ci-deepspeech-v2.yml

```diff
@@ -0,0 +1,32 @@
+name: Continuous Integration for Deepspeech Version 2
+on:
+  # Run on manual trigger
+  workflow_dispatch:
+
+  # Run on pull requests
+  pull_request:
+    paths-ignore:
+      - '*.md'
+
+  # Run when pushing to main or dev branches
+  push:
+    branches:
+      - main
+      - dev*
+
+  # Run scheduled CI flow daily
+  schedule:
+    - cron: '0 8 * * 0'
+
+jobs:
+  test_deepspeech_v2:
+    name: A job to test deepspeech version 2
+    runs-on: ubuntu-latest
+    container: minhitbk/art_testing_envs:deepspeech_v2
```

Comment:
Could you please check the CI log in step "Upload coverage to Codecov. It seems that the env does not have `curl` installed and cannot upload the coverage data: `==> curl not installed. Exiting.`

Reply:
Done.